src/hip/handlehip.cpp

```diff
@@ -81,21 +81,47 @@ std::size_t GetAvailableMemory()
 
 void* default_allocator(void*, size_t sz)
 {
-    if(sz > GetAvailableMemory())
-        MIOPEN_THROW("Memory not available to allocate buffer: " + std::to_string(sz));
-    void* result;
-    auto status = hipMalloc(&result, sz);
-    if(status != hipSuccess)
+    const auto available = GetAvailableMemory();
+    MIOPEN_LOG_I2("GetAvailableMemory " << available);
+    if(sz > available)
+        MIOPEN_LOG_I("GetAvailableMemory reports unsufficient memory to allocate " << sz);
+    void* ptr;
+    const auto status = hipMalloc(&ptr, sz);
+    if(status == hipSuccess)
     {
-        status = hipHostMalloc(&result, sz);
-        if(status != hipSuccess)
-            MIOPEN_THROW_HIP_STATUS(status,
-                                    "Hip error creating buffer " + std::to_string(sz) + ": ");
```

Comment:
@muralinr I do not think we want to append `+ ": "` to each `MIOPEN_THROW_HIP_STATUS`.